stable/application-mesh/templates/gateway.yaml

```diff
@@ -0,0 +1,48 @@
+{{- if .Values.global.mesh.enabled }}
+{{- range $_, $ingress := .Values.ingress }}
+{{- if ne $ingress.host "example.matic.com" }}
+{{- if or (eq $ingress.gateway "internal") (eq $ingress.gateway "external") }}
+{{- $host := required "Ingress host must be specified" $ingress.host  }}
+{{- $hosts := append ($ingress.additional_hosts | default $.Values.defaults.ingress.additional_hosts) $ingress.host }}
+---
+apiVersion: networking.istio.io/v1beta1
+kind: Gateway
+metadata:
+  name: {{ $host | replace "." "-" }}
+  namespace: istio-ingress
```

Comment:
I think when we create custom gateway for application - it should be created in application namespace